app/views/profiles/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<h2>Edit Profile </h2>
+<%= form_tag "/profile/#{@user.id}", method: :patch do  %>
+  <%= label_tag :zip_code %>
+  <%= text_field_tag :zip_code, "#{@user.zip_code}"%>
+
+  <%= submit_tag "Update Profile" %>
+<% end %>
+
+<p>Since you are signed in with google please update your name and e-mail address with them!</p>
```

Comment:
I'm a bit confused by this message

Reply:
I just threw something in fast. Wanted to convey that if they want to update thier e mail address/name do it with google, not us